default-recommendations/conditional-shortcuts-test.rkt

```diff
@@ -267,3 +267,9 @@ test: "cond with nested cond in last clause without else can't be flattened"
        [else
         (displayln "else")])]))
 ------------------------------
+
+test: "`if-else-false-to-and` is not refactorable when `and` is shadowed"
+------------------------------
+(define (and x y) (list x y))
+(if 'a (println "true branch") #f)
+------------------------------
```

Comment:
Can you add more tests, ideally with cases that you know will generate replacements that use things like `ORIGINAL-SPLICE` and `NEWLINE`?

Reply:
The existing tests generate replacements with those identifiers mixed into them, which is why I added that in the first place. I don't fully understand why though.

Reply:
Are the new tests I added enough?

Reply:
Yup. I just haven't had much chance to work on Resyntax lately.